### DISCUSSION TITLE
Confusing vertical alignment of list labels vs. Multiline MathML

### DISCUSSION THREAD

**Discussion by brichwin** [edrlab/thorium-reader#1794] [Ideas]
When an EPUB has list where the list items contain multi-line MathML, the list label appears vertically centered relative to the MathML instead of aligned with the first line of the rendered MathML. It can be confusing as to which parts of math are for which list items.

Here is a screenshot of one such instance (note how the single line math instances align with their list labels, but the multiline MathML items do not):
![image](https://user-images.githubusercontent.com/117550/185696870-cbf76fae-1294-4021-a7bf-b489cd010180.png)

**Comment by danielweck** [comment]
Thorium will not be able to fix this, as MathJax is responsible for MathML rendering, and ReadiumCSS + publisher CSS are responsible for the presentation / formatting styles. Moving to a discussion, as this may merit further fleshing out of possible solution (+ more detailed problem description)
